### DISCUSSION TITLE
Pubblicità nelle dashboard comunali

### DISCUSSION THREAD

**Discussion by pigreco** [opendatasicilia/informa-covid19#21] [Generale]
- È possibile inserirle?
- È lecito inserirle?
- Ha senso inserirle?
- Quali inserire?

grazie

**Comment by aborruso** [comment]
@pigreco  come mai te lo chiedi?

**Reply by pigreco** [reply]
@aborruso 


> @pigreco come mai te lo chiedi?

perché ho visto questa dashboard del comune di Serradifalco e c'è un banner pubblicitario:

https://datastudio.google.com/u/0/reporting/45ec94b1-32f0-489e-b882-977c12af81d4/page/0sn7B
![image](https://user-images.githubusercontent.com/7631137/116783657-32d89780-aa90-11eb-910f-c96b2b0e6645.png)

**Reply by dennisangemi** [reply]
La dashboard è stata duplicata dal webmaster de "la voce del nisseno" quindi quello credo sia una sorta di credit

**Reply by pigreco** [reply]
ok per la risposta alla prima domanda, rimane da rispondere alle ultime tre:

- È lecito inserirle?
- Ha senso inserirle?
- Quali inserire?

**Reply by aborruso** [reply]
Secondo me dobbiamo immaginare soltanto i credits di progetto da inserire. 

Poi ogni comune è "padrone" della dashboard.

Almeno in questa fase. Dopo la prima versione pubblica del progetto, potremmo immaginare di scrivere delle "raccomandazioni". 

Solo la mia opinione.

**Reply by pigreco** [reply]
IMHO, la dashboard e i dati sono `datibenecomune` e mettere pubblicità su pagine di questo tipo non lo trovo affatto opportuno.

m2c

**Reply by dennisangemi** [reply]
> Poi ogni comune è "padrone" della dashboard.

Sono d'accordo ma per il resto non saprei rispondere.

**Reply by giuragu** [reply]
Il banner messo così da l'idea del pubblicitario e non andrebbe bene. Se la dashboard è del comune potrebbe prevedersi una sezione ringraziamenti se vuole ringraziare qualcuno per la collaborazione (c'è da pensarci un attimo peró). Se invece la dashboard è di un privato/azienda, ad esempio, questi deve stare attento ad utilizzare loghi e nomi del comune come patrocinio (ogni comune ha regolamenti specifici in tal senso, e se non regolamentati peggio ancora). Solo un appunto veloce. Riflettiamoci su cosa meglio suggerire e come

**Reply by dennisangemi** [reply]
Come procedere allora?

**Reply by gpirrotta** [reply]
> * È possibile inserirle?

Come già detto da altri il comune diventa padrone della dashboard quindi può modificarla a suo piacimento

> * È lecito inserirle?

No, nella dashboard dovranno essere presente i credits del progetto. Se si vogliono fare dei ringraziamenti, come suggerito da @aborruso, possiamo dare delle indicazioni/raccomandazioni.

> * Ha senso inserirle?

Non ha molto senso, purtroppo possiamo fare affidamento solo al rispetto morale ed etico dei comuni (o di altri soggetti che vorranno riutilizzare il progetto).
